storage/innobase/handler/handler0alter.cc

```diff
@@ -8433,6 +8433,188 @@ innobase_update_foreign_cache(
 	DBUG_RETURN(err);
 }
 
+/** Changes SYS_COLUMNS.PRTYPE for one column.
+@param[in,out]	trx	transaction
+@param[in]	table_name	table name
+@param[in]	tableid	table ID as in SYS_TABLES
+@param[in]	pos	column position
+@param[in]	prtype	new precise type
+@return		boolean flag
+@retval	true	on failure
+@retval false	on success */
+static
+bool
+change_field_versioning_try(
+	trx_t* trx,
+	const char* table_name,
+	const ulint tableid,
```

Comment:
`tableid` should be `table_id_t` (64 bits). `ulint` is only 32 bits on 32-bit systems.

Reply:
That also caused a bb failure for windows.